gravitee-apim-e2e/lib/jest-utils.ts

```diff
@@ -125,4 +125,36 @@ export const describeIf = (condition) => (condition ? describe : describe.skip);
 export const describeIfV3 = describeIf(process.env.V4_EMULATION_ENGINE_DEFAULT == 'no');
 export const describeIfV4EmulationEngine = describeIf(process.env.V4_EMULATION_ENGINE_DEFAULT == 'yes');
 
+/**
+ * Check if debug tests should be skipped due to bridge compatibility.
+ * Debug tests are not compatible with client versions < 4.8.x when bridge server is >= 4.10.x
+ */
+function shouldSkipDebugTests(): boolean {
+  const clientTag = process.env.APIM_CLIENT_TAG;
+  if (!clientTag) {
+    return false;
```

Comment:
false -> true, because I suggest to reverse the logic of the function
```suggestion
    return true;
```